source/ed/connectors/fusio_parser.cpp

```diff
@@ -45,6 +45,60 @@ typename C::mapped_type get_object(const C& map, const std::string& obj_id, cons
     }
     return o;
 }
+/*
+Add default frame for all contributor without frame
+*/
+static void default_frames(GtfsData& gdata, Data& data){
+    for(auto contributor: data.contributors){
+        bool exist = false;
+        for(const auto frame : data.frames){
+            if (frame->contributor->uri == contributor->uri){
+                exist = true;
+                break;
+            }
+        }
+        if (! exist){
+            ed::types::Frame* to_return = new ed::types::Frame();
```

Comment:
`new` should be avoided :)
I'd like to suggest to use `auto to_return = make_unique<ed::types::Frame>();`